Cargo.toml

```diff
@@ -25,3 +25,12 @@ sha1 = "0.5"
 sha2 = { version = "0.7", default-features = false }
 tiny-keccak = "1.4"
 unsigned-varint = "0.3"
```

Comment:
We need the dependencies twice, once for our own tests, and once for the test feature that other crates can use from their dev-dependencies.